config/locales/en.yml

```diff
@@ -347,6 +348,14 @@ en:
       url: 'https://www.payments.service.gov.uk/'
       checkbox_label: Add payment links with reference numbers
       url_label: Your GOV.UK Pay payment link URL
+    save_and_return:
+      heading: 'Save for later'
+      lede: 'Give users the option of saving a partially completed form.'
+      description: 'Give users the option of saving a partially completed form for up to 28 days.'
+      page_lede: 'This places a save button at the bottom of each page of your form which will lead users through the save process. For details, see the %{href}'
+      href: <a class="govuk-link" href="%{url}" target="_blank" rel="noopener noreferrer">user guide section on save for later.</a>
+      url: 'https://moj-forms.service.justice.gov.uk/settings/'
```

Comment:
We wouldn't localise a url? This could just be hard coded in the view, or even better use <%= settings_path %>

Reply:
can do, will add the url to the above `href` key